regenerate-examples.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+# Setup requirements for executing jupyter notebooks
+python -m pip install -r requirements.txt
+python -m pip install -r requirements_docs.txt
+
+# Cleanup
+do_cleanup() {
+    echo "Performing cleanup of temporary resources"
+    rm -rf ./va-data-charlie
+}
+
+# Execute notebooks containing examples
+exec_examples() {
+    echo "Running notebooks containing examples"
+    for file in ./docs/source/content/examples/*.ipynb; do
```

Comment:
Is it always guaranteed that resulting files are sorted by name?

Reply:
https://github.com/vikinganalytics/mvg/pull/128/commits/8f150e5348dc597909c7c13b5f51e1284599b744 Fixed using the natural sort (ascending order) of `ls`.